html/sites/all/modules/custom/ocha_assessments/component/ocha-assessments-list.js

```diff
@@ -3739,29 +3739,19 @@ class OchaAssessmentsBase extends LitElement {
   }
 
   changeSrc(event) {
-    this.src = event.currentTarget.value;
```

Comment:
@attiks - not sure how this isn't already deployed - I see it as in dev but not in master branch - I think it's responsible for an error on staging:
"SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data"
e.g. https://dev.humanitarianresponse-info.ahconu.org/en/operations/ukraine/assessments/map

In the console, I get `Uncaught TypeError: URL constructor: ?f[0]=operation_id%3A43184 is not a valid URL.`

Reply:
Sorry, the issue was a typo introduced later. Confirming that this part all works as it should.